static/student-view.html

```diff
@@ -53,27 +53,32 @@
       </div>
     </div>
     <div id="learnocaml-exo-tab-buttons">
+      <button id="learnocaml-exo-button-list">Exercises</button>
       <button id="learnocaml-exo-button-stats" class="front-tab" disabled="">
         Stats
       </button>
-      <button id="learnocaml-exo-button-list">Exercises</button>
       <button id="learnocaml-exo-button-report">
         Report
       </button>
       <button id="learnocaml-exo-button-editor">Answer</button>
+      <button id="learnocaml-exo-button-draft">Draft</button>
       <button id="learnocaml-exo-button-text">Subject</button>
     </div>
     <div id="learnocaml-exo-tabs">
-      <div id="learnocaml-exo-tab-stats" class="front-tab"></div>
       <div id="learnocaml-exo-tab-list"></div>
+      <div id="learnocaml-exo-tab-stats" class="front-tab"></div>
+      <div id="learnocaml-exo-tab-report"></div>
       <div id="learnocaml-exo-tab-editor">
        <div id="learnocaml-exo-prelude"></div>
         <div id="learnocaml-exo-editor-pane" class="pane"></div>
         <div id="learnocaml-exo-editor-toolbar" class="buttons">
         </div>
       </div>
+      <div id="learnocaml-exo-tab-draft">
+        <div id="sync_time"></div>
```

Comment:
Just by convention, it's better to have hyphens (`-`) rather than underscores (`_`) in ids:

```suggestion
        <div id="learnocaml-sync-time"></div>
```